R/c14_date_list_calibrate.R

```diff
@@ -103,37 +65,34 @@ calibrate.c14_date_list <- function(x) {
     step_seq <- (step_width * (i - 1) + 1):if (i == amount_of_iterations)
     {nrow(calibrateable)} else {(step_width * i)}
 
-    # separate data in stacks of size step_width
-    calibrateable_step <- calibrateable[step_seq,]
-
-    # calibration
-    interval95[step_seq,] <- Bchron::BchronCalibrate(
-      ages      = calibrateable_step$c14age,
-      ageSds    = calibrateable_step$c14std,
-      calCurves = rep("intcal13", nrow(calibrateable_step)),
-      eps       = 1e-06
-    ) %>%
-      # extract border ages of the 2sigma range
-      plyr::ldply(., function(x) {
-        x[["densities"]]            %>% cumsum -> a      # cumulated density
-        which(a <= threshold)  %>% max    -> my_min # lower border
-        which(a > 1-threshold) %>% min    -> my_max # upper border
-        x[["ageGrid"]][c(my_min, my_max)]
-      }
-      )
+    # calibration in stacks of size step_width
+    calibrateable_prodistr[step_seq] <- calibrate_to_probability_distribution(
+      calibrateable[step_seq, ],
+      eps = 1e-06
```

Comment:
Should this be a static value? Should the user be able to chance this?

Reply:
Don't think so, not necessary